remove "AM_PROG_AR" from configure.in as it fails
--- a/configure.in
+++ b/configure.in
@@ -6,7 +6,6 @@
 AC_PROG_CC
 AC_PROG_CXX
 AC_PROG_INSTALL
-AM_PROG_AR
 AC_PROG_RANLIB
 AC_PROG_MAKE_SET
 AC_PATH_PROG(doxygen,doxygen)
